app/lib/rocketchat.js

```diff
@@ -74,7 +74,8 @@ const RocketChat = {
 
 	async getUserToken() {
 		try {
-			return await AsyncStorage.getItem(TOKEN_KEY);
+			const { serversDB } = database.databases;
+			return await serversDB.objects('servers').filtered('currentServer = true')[0].user;
```

Comment:
`return serversDB.objects('servers').filtered('currentServer = true')[0].user` no need for await when you are returning from function